src/identity/actions/creators/index.ts

```diff
@@ -35,3 +42,9 @@ export const setQuartzIdentityStatus = (status: RemoteDataState) =>
     type: SET_QUARTZ_IDENTITY_STATUS,
     status,
   } as const)
+
+export const setQuartzIdentityAccountName = (account: CurrentAccount) =>
```

Comment:
To avoid confusion we might want the function and the action type to be the same (set current identity account name versus set quartz identity account name).